queue/dispatcher.go

```diff
@@ -94,18 +100,35 @@ func (d *dispatcherImpl) Dispatch(ctx context.Context, job amboy.Job) error {
 				return
 			case <-ticker.C:
 				if err := job.Lock(d.queue.ID(), d.queue.Info().LockTimeout); err != nil {
+					grip.Debug(message.WrapError(err, message.Fields{
```

Comment:
Should we be nervous about noise caused by all of this logging?

Reply:
I wouldn't. The logs don't happen except in cases like server shutdown or somebody manually messing with the job document. The "slow job dispatching operation" already logs hundreds of times per minute.